fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -44,9 +44,9 @@ def execute(command, hide_output)
       end
 
       begin
-        FastlaneCore::FastlanePty.spawn(command) do |stdin, stdout, pid|
```

Comment:
When dealing with stdin/stdout and pipes, the naming is always confusing. The command's stdout is our stdin, and vice versa.

I think it's why the ruby API uses `r` and `w` for the `spawn` call (which we are wrapping for cross-platform reasons).

So we have to be clear at what we are naming here. Originally the first parameter was named `stdin` as it provided input to the block. Now it changes to `stdout` which means the command's stdout.

From a programming perspective what do we expect? Do we want to keep matching the spawn call's parameters? Do we want to name the parameter from the command or the block perspective?